CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -15,6 +15,19 @@ local SETTINGS_SHIELD_ACTIVE_POSITION = UDim2.new(0, 0, 0, 0)
 local SETTINGS_BASE_ZINDEX = 2
 local DEV_CONSOLE_ACTION_NAME = "Open Dev Console"
 
+local BUMPER_IMG_WIDTH = 115
+local BUMPER_IMG_HEIGHT = 64
+local BUMPER_MARGIN = 30
+local MINIMUM_BUMPER_HINT_HEIGHT = 15
+
+local BUMPER_LEFT_SPRITE_OFFSET = Vector2.new(116, 416)
```

Comment:
There is indeed. The question is, do we want to include it with the CoreScripts or what?